app/integration_test/README.md

```diff
@@ -1,211 +1,53 @@
-# Integrations- / E2E-Tests via `integration_test`
+# Integration Tests
+Unit tests and widget tests are handy for testing individual classes, functions, or widgets. However, they generally don’t test how individual pieces work together as a whole, or capture the performance of an application running on a real device. These tasks are performed with integration tests.
 
-Dieser Ordner wird für Tests mit dem [`integration_test`](https://pub.dev/packages/integration_test) Package benutzt.
+Integration tests are written using the [integration_test](https://github.com/flutter/flutter/tree/master/packages/integration_test) package, provided by the SDK.
 
-Das `integration_test`-Package ist die "verbesserte" Version von `flutter_driver` und ermöglicht es einem mit der `widgetTest`-API E2E- bzw. Integrations-Tests zu schreiben, welche auf einem Simulator, echten Gerät oder sogar dem ["Firebase Test Lab"](https://firebase.google.com/docs/test-lab) laufen.
+## How to run integration tests
+In order to run the integration tests, you need to setup a fresh Sharezone account which can be used for the integration tests. This account needs to be linked to an email address and a password.
 
-Die Tests, Integration von `integration_test` und eine richtige Pipeline sind momentan noch WIP.    
-Es funktionieren momentan schon mal das erfolgreiche Ausführen der Platzhalter Integrations-Tests auf:
-* Android (via Flutter-Driver oder `gradlew`)
-* Web
-* macOS
-
-### Flutter Driver
-Mithilfe folgenden Befehls kann der `foo_test.dart` lokal laufen gelassen werden.   
-Der Befehl muss aus dem `app`-Ordner heraus laufen gelassen werden (damit die Pfad übereinstimmen).
-```
-flutter drive \
-  --driver=test_driver/integration_test.dart \
-  --target=integration_test/foo_test.dart \
-  --flavor prod
-```
-
-Der `foo_test.dart` **soll extra fehlschlagen** (er beinhaltet einen erfolgreichen und fehlschlagen Test-Fall).
-
-<details>
-  <summary>Beispiel-Output (hier wird mehrmals das gleiche ausgeprintet warum auch immer)</summary>
-  
-```
```

Comment:
@Jonas-Sander I removed the old `README.md` for the integration tests because they were only just the documentation of how to set up the integration tests which is already good documented in the `README.md` of the `integration_test` package: https://github.com/flutter/flutter/tree/master/packages/integration_test